components/dashboard/src/App.tsx

```diff
@@ -274,6 +281,32 @@ function App() {
         }
     }, [user]);
 
+    const checkCookie = () => {
+        if (isGitpodIo()) {
+            if (Cookies.get(COOKIE_NAME) === undefined) {
```

Comment:
This code would read easier in a form a-la:
```
const consent = ConsentCookie.get(domain);
if (consent === undefined) {
   showCookieBanner();
}
```
Everything else should be an implementation detail.

Reply:
Thanks! Definitely better :)